src/tests/map/all.mk

```diff
@@ -19,11 +19,17 @@ $(MAP_OUTPUT): $(MAP_UNIT_BIN) | $(BUILD_DIR)/tests/map/
 #
 $(BUILD_DIR)/tests/map/%.out: $(top_srcdir)/src/tests/map/%
 	${Q}echo MAP_TEST $(notdir $<)
-	${Q}if ! $(MAP_UNIT) -d $(top_srcdir)/raddb -D $(top_srcdir)/share/dictionary $< > $@ 2>&1; then \
+	${Q}rm -f "$@.ok"
+	${Q}if ! $(MAP_UNIT) -d $(top_srcdir)/raddb -D $(top_srcdir)/share/dictionary -r "$@.ok" $< > $@ 2>&1; then \
+		if [ ! -f "$@.ok" ]; then \
```

Comment:
This pretty much entirely negates the point of writing out the 'ok' files.  You need to add this to the test at the top i.e.

```
${Q}if ! $(MAP_UNIT) -d $(top_srcdir)/raddb -D $(top_srcdir)/share/dictionary -r "$@.ok" $< > $@ 2>&1 || [ ! -f "$@.ok" ]
```

Reply:
The example command that's written out on test failure also needs to be updated with the -r flag